### ISSUE TITLE
iPhone 8, 8+, X

### ISSUE THREAD

**Issue by alvincrisuy** [lmirosevic/GBDeviceInfo#68]
Device Information for iPhone 8, 8+, and X

**Comment by sushant40** [comment]
eager to know this myself.

**Comment by alvincrisuy** [comment]
iPhone10,1 iPhone 8
iPhone10,2 iPhone 8 Plus
iPhone10,3 iPhone 8
iPhone10,4 iPhone 8 Plus
iPhone10,5 iPhone X
iPhone10,6 iPhone X

I'm not sure if this is the right one but I saw a link that says the ff info.

**Comment by mobinzk** [comment]
The correct device information are:
```
            case "iPhone10,1", "iPhone10,4":                 return .iPhone8
            case "iPhone10,2", "iPhone10,5":                 return .iPhone8Plus
            case "iPhone10,3", "iPhone10,6":                 return .iPhoneX```

**Comment by lmirosevic** [comment]
Merged in #70 and pushed to CocoaPods as `4.4.0`
